dev/build/datatracker-start.sh

```diff
@@ -3,8 +3,13 @@
 echo "Running Datatracker checks..."
 ./ietf/manage.py check
 
-echo "Running Datatracker migrations..."
-./ietf/manage.py migrate --skip-checks --settings=settings_local
+if ! ietf/manage.py migrate --skip-checks --check ; then
+    echo "Unapplied migrations found, waiting to start..."
+    sleep 5
+    while ! ietf/manage.py migrate --skip-checks --check ; do 
```

Comment:
maybe echo something here so we know if were just stuck waiting for migrations?

Reply:
Actually, right after the `if` we `echo Starting datatracker...` so I think that's ok? I'll add a similar one to Celery since it doesn't have one